code/game/objects/items/devices/flashlight.dm

```diff
@@ -133,6 +133,9 @@
 	icon_state = "lampgreen"
 	item_state = "lampgreen"
 
+/obj/item/flashlight/lamp/green/off/Initialize()
+	extinguish_light()
```

Comment:
Again, just make the definition have `off = FALSE`

Reply:
I thought there would be an icon issue if I did that, but nope. Set it to `on = FALSE` and works properly